easybuild/easyconfigs/t/Tk/Tk-8.6.7-intel-2017b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.7'
```

Comment:
@boegel Is it worth indicating `X11` in the suffix?

Reply:
We haven't done so in the past, I would stick to that.

Generally speaking, we only include dep versions in the `versionsuffix` for one of two reasons:
* it's a very common dependency, and the `versionsuffix` helps to avoid loading modules that depend on different versions together (example: Python, Perl)
* there's already an easyconfig file with the same name/version/toolchain (incl. version), and you're creating some kind of variant where you change the version of one or more dependencies (commonly to ensure compatibility with something else)